package.json

```diff
@@ -28,16 +28,14 @@
   "scripts": {
     "compile": "microbundle build -f modern,umd --globals react=React",
     "postcompile": "cp dist/index.modern.mjs dist/index.modern.js && cp dist/index.modern.mjs.map dist/index.modern.js.map",
-    "test": "run-s eslint tsc-test jest",
-    "eslint": "eslint --ext .js,.ts,.tsx .",
-    "jest": "jest",
+    "test": "run-s eslint tsc-test vitest playwright",
```

Comment:
I see, we run playwright in "test". If we want to keep it, we can't separate workflows.